yang/frr-routing.yang

```diff
@@ -0,0 +1,148 @@
+module frr-routing {
+  yang-version "1.1";
+  namespace "http://frrouting.org/yang/routing";
+  prefix "rt";
+
+  import ietf-yang-types {
+    prefix "yang";
+  }
+
+  import frr-vrf {
+    prefix frr-vrf;
```

Comment:
not necessary?

Reply:
[RFC 7950](https://tools.ietf.org/html/rfc7950#section-7.1.5):
>    The mandatory "prefix" substatement assigns a prefix for the imported
>     module that is scoped to the importing module or submodule.

it's necessary :/